docs/getting-started/README.md

```diff
@@ -0,0 +1,45 @@
+# Getting Started with Declarative Gradle
+
+Declarative Gradle is in the experimental stage,
+so it requires some extra steps to get started with.
+It has limited testing and compatibility with IDEs,
+so your mileage may vary.
+
+## Installing Components
+
+- You will need the latest [Gradle Build Tool nightly build](https://gradle.org/nightly/).
```

Comment:
🤔 I would make this disclaimer even more prominent.  I try to keep the wrapper up-to-date, but sometimes miss it or it isn't immediately possible (waiting on reviews).  I'd make this a callout and note that running this project frequently requires bleeding edge features and experiments only available in the nightly, and that you might have to manually download it, etc.

Reply:
Agreed, but I do not think it should be in getting started, which is a read-once thing. I would add it to the generic user guide if you're fine with that